GruntFile.js

```diff
@@ -17,18 +17,27 @@ module.exports = function (grunt) {
             commitMessage: 'Release <%= version %>', //default: 'release <%= version %>'
             tagMessage: 'Tag version <%= version %>' //default: 'Version <%= version %>'
          }
+      },
+      
+      copy: {
+        typings: {
+            files: [
+                { expand: true, cwd: './typings', src: ['./**'], dest: '.' }
```

Comment:
For my reference - please can you let me know why the `.d.ts` are copied into the root of the project as part of the release. As they are untracked, is this just to have them available in npm?